x/evm/types/call_type.go

```diff
@@ -0,0 +1,10 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+package types
+
+type CallType int
+
+const (
+	RPC CallType = iota + 1
```

Comment:
add comment here when would be an RPC call
```suggestion
        // RPC call type is used on requests to eth_estimateGas rpc API endpoint
	RPC CallType = iota + 1
	// Internal call type is used in case of smart contract methods calls
```

Reply:
done